addons/mrp/models/mrp_unbuild.py

```diff
@@ -44,7 +44,7 @@ class MrpUnbuild(models.Model):
         required=True, states={'done': [('readonly', True)]}, check_company=True)
     mo_id = fields.Many2one(
         'mrp.production', 'Manufacturing Order',
-        domain="[('state', 'in', ['done', 'cancel']), ('company_id', '=', company_id)]",
+        domain="[('state', '=', 'done'), ('product_id', '=', product_id), ('company_id', '=', company_id)]",
```

Comment:
On the previous version if no product was selected all MOs would be available (if done and in the correct company), now only those without a product can be selected. Is that intended?

Reply:
it's not
bad @Whenrow